druid/src/widget/label.rs

```diff
@@ -106,22 +106,9 @@ impl<T: Data> Label<T> {
         Label::new(text)
     }
 
-    /// Set a new text.
-    ///
-    /// Takes an already resolved string as input.
-    ///
-    /// If you're looking for full [`LabelText`] support,
-    /// then you need to create a new [`Label`].
-    ///
-    /// [`Label`]: #method.new
-    /// [`LabelText`]: enum.LabelText.html
-    pub fn set_text(&mut self, text: impl Into<String>) {
-        self.text = LabelText::Specific(text.into());
-    }
-
     /// Set text alignment.
     #[deprecated(since = "0.5.0", note = "Use an Align widget instead")]
-    pub fn text_align(self, _align: UnitPoint) -> Self {
```

Comment:
being as this is deprecated, I wouldn't bother renaming it; the point of the deprecation is to avoid breaking existing code, while pointing to the new API.

(I'm not worried about breaking these other methods; they are relatively new and I don't think they are widely used. You could do this deprecation trick for fun if you wanted, but i'm not concerned)